src/burp/UuidIssue.java

```diff
@@ -19,37 +19,43 @@ public UuidIssue(IHttpRequestResponse baseRequestResponse,
 	}
 
 	@Override public String getIssueName() {
-		return "Version " + uuid.version() + " UUID in HTTP request";
+		int v = uuid.version();
+		if (v > 5 || v < 1)
+			return "Unknown version " + v + " UUID in HTTP Request";
+		else
+			return "Version " + v + " UUID in HTTP request";
 	}
 
 	@Override public String getIssueDetail() {
 		int v = uuid.version();
-		final String prefix = "The request contains the version " + v +
+		final String known_prefix = "The request contains the version " + v +
```

Comment:
Let's be consistent (at least within a single class) and use the CamelCase version (`knownPrefix`) -- or as you can see my other comment below, you could even just use the original name (`prefix`) if you remove `unknown_prefix`.